Makefile

```diff
@@ -26,7 +26,7 @@ pull:
 	docker pull ${IMAGE}
 
 fetch:
-	curl ${SPEC_URL} | jq . > ${SPEC_FETCHED_FILE}
+	cp ../packethost-api/docs/openapi/public_openapi3.json ${SPEC_FETCHED_FILE}
```

Comment:
I think the upstream now includes this version of the API spec. Is it safe to remove this customization now and move the PR from draft to ready?

Reply:
Yep, done.